client/package.json

```diff
@@ -97,17 +98,6 @@
       "/public/"
     ]
   },
-  "husky": {
```

Comment:
Shouldn't we remove `husty` and `lint-staged` from dev deps as well?

Reply:
Removed unused deps in https://github.com/Sketch-sh/sketch-sh/pull/323/commits/11dda1d2dbc781af8ef3480bdd3b1c452a7e2d69